content/general/versions/v0.8/webhooks.textile

```diff
@@ -190,15 +225,86 @@ h4. Example @channel.presence@ @update@ and @leave@ JSON payload
 
 h2(#messages). Messages
 
-Receiving messages via WebHooks is currently in beta. As such, this functionality is currently only available to Enterprise customers. If you are interested in receiving messages via WebHooks, please "get in touch with us":https://www.ably.io/contact.
+When a message is published on a channel, a message WebHook event is triggered.
+
+Every WebHook request will contain one or more enveloped messages in the root level attribute @items@, each of which in turn contains one or messages. See "WebHook transport details":#webhook-transport below to find out more about batching and expected throughput.
+
+Each message envelope in the @items@ array consists of a JSON-like object with the following attributes:
```

Comment:
Looks like that change happened in the spec, but not the webhook docs: https://github.com/ably/docs/commit/f960efe15a641efcb76de481971dc142abde546f . I'll get rid of it. (got added here just cause it was already "json-like" in the 0.9 version, and I copied the whole section to the 0.8 version)